src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx

```diff
@@ -1,61 +1,63 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import useAppManager from '@site/src/hooks/useAppManager';
-import SwipeableBottomSheet from '@site/src/components/SwipeableBottomSheet';
-import { Heading } from '@deriv-com/quill-ui';
-import './app-register-success-modal.scss';
+import { Heading, Modal } from '@deriv-com/quill-ui';
+import useDeviceType from '@site/src/hooks/useDeviceType';
+import '../../Dialogs/DeleteAppDialog/delete-app-dialog.scss';
 
 interface IAppRegisterSuccessModalProps {
   onConfigure: () => void;
   onCancel: () => void;
-  is_desktop: boolean;
 }
 
 export const AppRegisterSuccessModal = ({
   onConfigure,
   onCancel,
-  is_desktop,
 }: IAppRegisterSuccessModalProps) => {
   const { app_register_modal_open } = useAppManager();
+  const { deviceType } = useDeviceType();
+
+  useEffect(() => {
+    if (app_register_modal_open) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
+    }
+  });
```

Comment:
@agrim-deriv as discussed let's create a custom hook and use it in all the necessary places.